app/views/katello/subscription_mailer/subscription_expiry.html.erb

```diff
@@ -40,6 +40,7 @@
         </tr>
       <% end %>
     </table>
+  <%= link_to _("Report"), report_url %>
```

Comment:
On my dev box the `report_url` seems to use `http` and not `https`, which causes errors for me.  When I manually change the url to use `https`, it works.  Not sure if this will be an issue in production.. @jturel ?

Reply:
Should be fine - though not sure why Setting[:foreman_url] is using http in dev